andela/mrm_api
# #164135004 Document Backend API codebase

### What does this PR do?
Adds descriptions to the Graphql queries and mutations.
### Description of the task to be completed?
When one visits the documentation they should be able to see the respective descriptions of both the mutations and the queries.
### How should this be manually tested?
 - Clone the branch and follow the following steps to [setup](https://github.com/andela/mrm_api).
 - Checkout by running `git checkout ft-documentation-164135004`
 - Go to the docs tab on the top right of the browser to see the descriptions of various queries and mutations.

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#164135004](https://www.pivotaltracker.com/story/show/164135004)
### Screenshots
 - #### Description of a mutation that creates a floor
![image](https://user-images.githubusercontent.com/29709981/53790630-7a0e8080-3f38-11e9-85a7-08416cab3e9f.png)

- #### Description of a query to get a room
![image](https://user-images.githubusercontent.com/29709981/53790641-81ce2500-3f38-11e9-8e5f-f91da1907ebf.png)
